api/routes/qaris.js

```diff
@@ -42,4 +44,68 @@ router.get('/related/:id', (req, res) => {
     .then(related => res.send(related));
 });
 
+router.get('/:id/download', (req, res) => {
+  const archive = archiver('zip', {
+    zlib: { level: 9 } // Sets the compression level.
+  });
+  const header = {
+    'Content-Type': 'application/x-zip',
+    Pragma: 'public',
+    Expires: '0',
+    'Cache-Control': 'private, must-revalidate, post-check=0, pre-check=0',
+    'Content-disposition': `attachment; filename=\"complete_quran.zip\"`,
+    'Transfer-Encoding': 'chunked',
+    'Content-Transfer-Encoding': 'binary'
+  };
+
+  archive
+    .on('warning', err => {
+      if (err.code === 'ENOENT') {
+        console.log(err);
```

Comment:
remove?

Reply:
Still a js newbie, what's the appropriate way of logging non-fatal errors?